src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/ProblemDetailsClientErrorFactory.cs

```diff
@@ -28,6 +29,7 @@ public IActionResult GetClientError(ActionContext actionContext, IClientErrorAct
             {
                 problemDetails.Title = errorData.Title;
                 problemDetails.Type = errorData.Link;
+                problemDetails.Extensions[RequestIdentifierKey] = actionContext.HttpContext.TraceIdentifier;
```

Comment:
@glennc - did you track this down? Is this what we're supposed to use?

Reply:
Ran this by @glennc - I updated this to use the same pattern as the Error handler which is to use `Activity.Current` if available